diff --git a/src/App/Features/Context/FeatureContext.php b/src/App/Features/Context/FeatureContext.php
index 79deb7f..0d7cc29 100644
--- a/src/App/Features/Context/FeatureContext.php
+++ b/src/App/Features/Context/FeatureContext.php
@@ -2,6 +2,7 @@
 
 namespace App\Features\Context;
 
+use App\Entity\Team;
 use Behat\Behat\Exception\PendingException;
 use Behat\Gherkin\Node\PyStringNode;
 use Behat\Gherkin\Node\TableNode;
@@ -14,6 +15,7 @@ use Symfony\Component\DependencyInjection\ContainerInterface;
 use Behat\Behat\Context\Step\Given;
 use Behat\Behat\Context\Step\When;
 use Behat\Behat\Context\Step\Then;
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
 
 require_once __DIR__.'/../../../../vendor/phpunit/phpunit/PHPUnit/Autoload.php';
 require_once __DIR__.'/../../../../vendor/phpunit/phpunit/PHPUnit/Framework/Assert/Functions.php';
@@ -23,6 +25,43 @@ class FeatureContext extends MinkContext implements KernelAwareInterface
     private $kernel;
 
     /**
+     * @Given /^there are (\d+) teams$/
+     */
+    public function thereAreTeams($num)
+    {
+        $faker = $this->createFaker();
+        for ($i = 0; $i < $num; $i++) {
+            $team = new Team();
+            $team->setName($faker->name);
+            $team->setDescription($faker->paragraph());
+
+            $this->getEntityManager()->persist($team);
+            $this->getEntityManager()->flush();
+        }
+    }
+
+    /**
+     * @When /^I click "([^"]*)"$/
+     */
+    public function iClick($linkName)
+    {
+        return new When(sprintf('I follow "%s"', $linkName));
+    }
+
+    /**
+     * @Then /^I should see (\d+) rows in the table$/
+     */
+    public function iShouldSeeRowsInTheTable($rowCount)
+    {
+        $table = $this->getPage()->find('css', 'table');
+        assertNotNull($table, 'Cannot find a table!');
+
+        $rows = $table->findAll('css', 'tbody tr');
+
+        assertCount(intval($rowCount), $rows);
+    }
+
+    /**
      * Sets Kernel instance.
      *
      * @param KernelInterface $kernel HttpKernel instance
@@ -65,4 +104,15 @@ class FeatureContext extends MinkContext implements KernelAwareInterface
     {
         return $this->getContainer()->get('doctrine.orm.entity_manager');
     }
+
+    /**
+     * Clears the database before every scenario
+     *
+     * @BeforeScenario
+     */
+    public function clearDatabase()
+    {
+        $purger = new ORMPurger($this->getEntityManager());
+        $purger->purge();
+    }
 }
diff --git a/src/App/Features/team.feature b/src/App/Features/team.feature
new file mode 100644
index 0000000..d1afcd4
--- /dev/null
+++ b/src/App/Features/team.feature
@@ -0,0 +1,10 @@
+Feature: Team management
+  In order to view and control which teams are in the system
+  As an admin
+  I need to be able to add/edit/delete teams
+
+  Scenario: Listing teams
+    Given there are 5 teams
+    And I am on "/"
+    When I click "Teams"
+    Then I should see 5 rows in the table
