diff --git a/src/App/Resources/views/Team/index.html.twig b/src/App/Resources/views/Team/index.html.twig
index a80b4c5..ac388c0 100644
--- a/src/App/Resources/views/Team/index.html.twig
+++ b/src/App/Resources/views/Team/index.html.twig
@@ -6,6 +6,7 @@
             <tr>
                 <th>Name</th>
                 <th># Players</th>
+                <th>Captain</th>
                 <th>&nbsp;</th>
             </tr>
         </thead>
@@ -14,6 +15,7 @@
                 <tr>
                     <td>{{ team.name }}</td>
                     <td>{{ team.players|length }}</td>
+                    <td>{{ team|captain }}</td>
                     <td>
                         <a href="{{ path('app_team_show', {'id': team.id}) }}">Show</a>
                     </td>
diff --git a/src/App/Twig/AppExtension.php b/src/App/Twig/AppExtension.php
new file mode 100644
index 0000000..609cdea
--- /dev/null
+++ b/src/App/Twig/AppExtension.php
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Twig;
+
+use App\Entity\Team;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class AppExtension extends \Twig_Extension implements ContainerAwareInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * Returns the name of the extension.
+     *
+     * @return string The extension name
+     */
+    public function getName()
+    {
+        return 'app';
+    }
+
+    public function getFilters()
+    {
+        return array(
+            new \Twig_SimpleFilter('captain', array($this, 'getTeamCaptain'))
+        );
+    }
+
+    /**
+     * @param Team $team
+     * @return \App\Entity\Player
+     */
+    public function getTeamCaptain(Team $team)
+    {
+        $playerRepo = $this->container->get('app.entity.player_repository');
+
+        return $playerRepo->getCaptainForTeam($team);
+    }
+
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+} 
\ No newline at end of file
